README.md

```diff
@@ -45,9 +45,17 @@ are being created:
 # Cleanup
 
 In cases where the beaker process is killed before finishing, it may leave resources in Hetzner cloud. These will need to be manually deleted.
-
 Look for servers in your project named exactly as the ones in your beaker host configuration and SSH keys with names beginning with `Beaker-`.
 
+Every created cloud instance gets a label `delete_vm_after: 1698792887`. By
+default this is the unit timestamp during VM creation + an hour.
+
+You can modify the default of an hour by setting the `BEAKER_HCLOUD_DELETE_VM_AFTER`
+environment variable to any positive integer. It will be interpreted as hours.
+
+It's up to the user to scan for VMs where the `delete_vm_after` points to a
+past timestamp and delete them.
```

Comment:
In a normal workflow VMs are deleted right? Only where the was some timeout are they not deleted? It's mentioned in the first sentence, but perhaps worth to make it explicit here.

Reply:
yes. I rephrased it so it's hopefully more clear.